app/controllers/api/api_controller.rb

```diff
@@ -1,7 +1,7 @@
 module Api
   class ApiController < Api::BaseController
     def options
-      head(:ok)
+      render_resource :product, product_info
```

Comment:
we need this under a product_info element, maybe just render :json => { :product_info => product_info }

Reply:
Makes sense, I need 3 more items `Settings.server.custom_logo`, `Settings.server.custom_login_logo` and `Settings.server.custom_brand` (all boolean values). Do you have any idea how to structure them? Or is it okay to add it into `product_info`?

Reply:
product_info is more geared toward MiQ itself, above may fit better under server_info, or maybe even a new section like brand_info.

Reply:
Maybe we can make use of https://github.com/ManageIQ/manageiq/pull/17755 for the version info?  I'm not sure what the rest of the information is.
Why do we want to return booleans for things like `Settings.server.custom_logo`? (I don't know how it's used)  Wouldn't it make more sense to have a `logo` key with a value of the href to the image or something like that?

Reply:
@bdunne we want them booleans because they are already booleans, in the OPS UI there's a checkbox for using the custom image. The image itself is on a fixed location, so the filename is static for all the three cases. So historical reasons, that's why :)

Reply:
@abellotti I think I'll go with the `brand_info`, so
```ruby
{
  :product_info => product_info,
  :brand_info => ...
}
```